### PR TITLE
Fixes #11216 - use rails-i18n gem for translating rails strings

### PR DESCRIPTION
**Pull request by tbrisker**

### CODE REVIEW

File: Gemfile [hunk 1]
```diff
@@ -25,6 +25,7 @@ gem 'safemode', '~> 1.2'
 gem 'fast_gettext', '~> 0.8'
 gem 'gettext_i18n_rails', '~> 1.0'
 gem 'i18n', '~> 0.6.4'
+gem 'rails-i18n', '~> 3.0.0'
```
Comment (lzap) [inline Gemfile:28]: Wild idea - can we make '3.0.0' a constant like `RAILS_VERSION` so it's more obvious? Maybe we have more gems which are in sync with Rails version, couldn't find any tho.

Comment (tbrisker) [inline Gemfile:28]: The only other gem I found using this scheme is audited-activerecord, and nobody is promising us they will keep the same, which might lead to head-scratching-type-breakage.

Comment (lzap) [inline Gemfile:28]: Ok better to leave it as is.

**Comment by domcleal** [comment]
Merged as 60f8b29, thanks @tbrisker.

**Pull request closed without merging**